python/orca/example/learn/openvino/predict.py

```diff
@@ -55,7 +55,7 @@ def crop(img, w, h):
 
     if args.cluster_mode == "local":
         init_orca_context(cores=args.core_num, memory=args.memory)
-    elif args.cluster_mode == "yarn":
+    elif args.cluster_mode.starswith("yarn"):
```

Comment:
starts

Reply:
Sure, it's better to put orca test scripts in orca/dev/example